test/extended/images/imagechange_buildtrigger.go

```diff
@@ -311,8 +311,10 @@ func runTest(t g.GinkgoTInterface, oc *exutil.CLI, testname string, projectAdmin
 
 		// the first tag did not have an image id, so the last trigger field is the pull spec
 		expectedLastTriggerTag := registryHostname + "/openshift/test-image-trigger:" + tag
-		lastTriggeredImageId := updatedConfig.Spec.Triggers[0].ImageChange.LastTriggeredImageID
-		o.Expect(lastTriggeredImageId).To(o.Equal(expectedLastTriggerTag))
+		lastTriggeredImageIdSpec := updatedConfig.Spec.Triggers[0].ImageChange.LastTriggeredImageID
+		lastTriggeredImageIdStatus := updatedConfig.Status.ImageChangeTriggers[0].LastTriggeredImageID
+		o.Expect(lastTriggeredImageIdStatus).To(o.Equal(expectedLastTriggerTag))
+		o.Expect(lastTriggeredImageIdSpec).To(o.BeEmpty())
```

Comment:
I think you have a "chicken or egg" situation here @alicerum 

1) you are trying to merge your openshift-apiserver PR where you only set status
2) but that is no merging because the current version of this test attempt to verify spec
3) but this test is currently failing in this PR because you openshift-apiserver PR which ensures spec is not set has not merged

I think you will need to update this test in 2 PRs.
a) change this test first so you only verify what status is set to
b) get your openshift-apisever PR to merge with the test update from 1)
c) come back here in a new PR and add the verification that spec is empty

Reply:
Yeah, i think it is the right way to go :) Will probably go for that for now, thank you